bigbluebutton-html5/imports/ui/components/breakout-room/create-breakout-room/queries.ts

```diff
@@ -69,7 +69,23 @@ export const getBreakoutCount = gql`
   }
 `;
 
+export const getLastBreakout = gql`
+subscription {
+  user {
+    lastBreakoutRoom {
+      breakoutRoomId
+      currentlyInRoom
+      isDefaultName
+      sequence
+      shortName
+      userId
+    }
+  }
+}
+`;
```

Comment:
is it possible to used a `query` instead of `subscription` ?
I guess the values won't change during the creation of a breakout room.

Moreover, maybe you could filter users who has a last breakout room:

```gql
query {
  user(where: {lastBreakoutRoom: {breakoutRoomId: {_is_null: false}}}) {
    userId
    lastBreakoutRoom {
      breakoutRoomId
      currentlyInRoom
      isDefaultName
      sequence
      shortName
    }
  }
}

```

Reply:
Actually you need to double check if it is necessary to have a new query/subscription only for this function. 
This screen is already fetching all users...

Reply:
I see  fields like `isDefaultName` being fetched but it is never used in the code.
If it will not be used it shouldn't be fetched.

But I think it is necessary in 2.7 logic, are you changing the logic used to re-assign users?